ubuntu_requirement_installation.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+set -ex
+
+source lib/logging.sh
+
+# sudo apt install -y libselinux-utils
+# if selinuxenabled ; then
+#     sudo setenforce permissive
+#     sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
+# fi
+
+# Update to latest packages first
+sudo apt -y update
+
+# Install EPEL required by some packages
+# if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
+#     if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+#         sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/epel-release-7-11.noarch.rpm
+#     else
+#         sudo yum -y install epel-release --enablerepo=extras
+#     fi
+# fi
+
+# Work around a conflict with a newer zeromq from epel
+# if ! grep -q zeromq /etc/yum.repos.d/epel.repo; then
+#   sudo sed -i '/enabled=1/a exclude=zeromq*' /etc/yum.repos.d/epel.repo
+# fi
+
+# Install required packages
+# python-{requests,setuptools} required for tripleo-repos install
+sudo apt -y install \
+  net-tools \
+  crudini \
+  curl \
+  dnsmasq \
+  figlet \
+  golang \
+  nmap \
+  patch \
+  psmisc \
+  python-pip \
+  python-netaddr \
+  python-requests \
+  python-setuptools \
+  python-libvirt \
+  wget
+
+# Check if 'ifconfig' is available!
+if [[ ! $(ifconfig) ]]; then
+  echo "Cannot run ifconfig..."
+  echo "Install net-tools package"
+  sudo apt -y install net-tools
+fi
+
+# We're reusing some tripleo pieces for this setup so clone them here
+
+## We don't need TripleO repos in Ubuntu
+##cd
+##if [ ! -d tripleo-repos ]; then
+##  git clone https://git.openstack.org/openstack/tripleo-repos
+##fi
+##pushd tripleo-repos
+##sudo python setup.py install
+##popd
+
+# Needed to get a recent python-virtualbmc package
+#sudo tripleo-repos current-tripleo
+
+# There are some packages which are newer in the tripleo repos
+
+# Setup yarn and nodejs repositories
+#sudo curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+#curl -sL https://rpm.nodesource.com/setup_10.x | sudo bash -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
+# Add this repository to install podman
+sudo add-apt-repository -y ppa:projectatomic/ppa
+
+# Update some packages from new repos
+sudo apt -y update
+
+# make sure additional requirments are installed
+
+##No bind-utils. It is for host, nslookop,..., no need in ubuntu
+sudo apt -y install \
+  ansible \
```

Comment:
The ansible installed this way has issues. We should probably go with virtual environments